vss-extension.json

```diff
@@ -1,10 +1,10 @@
 {
     "manifestVersion": 1,
-    "id": "DefenderForDevops-PipelineDecorators",
+    "id": "ms-dfd-code-to-cloud",
     "publisher": "ms-securitydevops",
-    "version": "0.0.2",
-    "name": "DefenderForDevops-PipelineDecorators",
-    "description": "ADO Pipeline Decorator Extension for Defender for Devops",
+    "version": "0.0.3",
+    "name": "Microsoft Defender for DevOps - Code to cloud",
+    "description": "Code to cloud extension for Microsoft Defender for DevOps",

```

Comment:
This description should still include the fact that this is an extension for ADO.